### DISCUSSION TITLE
移动联通设墙导致无法正版登录

### DISCUSSION THREAD

**Discussion by lactobionicAcid** [Meloong-Git/PCL#2738] [讨论]
> [!IMPORTANT]
> **请勿将该问题作为 Issue 反馈给龙猫。**
该问题与 PCL 启动器无关，同时龙猫也不具有修复该问题的能力。

> [!IMPORTANT]
> 有关因服务故障造成的无法正版登录，请移步 #4147。

![image](https://github.com/Hex-Dragon/PCL2/assets/115075633/78d54ab3-bb43-4d68-9251-607a2bfad466)

2023 年国庆节当天，国家反诈中心、工信部反诈中心联合移动和联通两大运营商，以未知原因封禁 api.minecraftservices.com 和二级域名为 mojang.com 的地址（重定向至提示页面）。届时，中国移动和中国联通网络均无法正常正版登录。

如果你很不幸被两大运营商拦截在外，则有以下几种解决方法：

- 修改 DNS 服务器：
    - 将你的 DNS 服务器修改为未受限制的 DNS 服务器，在网络上直接搜索 “公共 DNS” 应该能搜到一大堆，从中随便选一两个即可。详细的修改方法请自行百度或翻阅 PCL2 的内置帮助。
    - 之后，按下 `Windows` + `R` 键运行指令 `ipconfig /flushdns` 以刷新 DNS 缓存。可能需要重启 PCL2 启动器或电脑。
- 使用电信流量上网；
- 使用[某种以 7 根木棍合成的方块](https://www.mcmod.cn/item/39.html)。

如果上述方法仍无法解决问题，那么...... _不妨先用离线模式玩一段时间，等到封禁结束后再说吧。(´-ω-`)_
_当然这个问题或许不会再发生了的说。_

**Comment by SALTWOOD** [comment]
补充：目前国内80余个节点的数据显示，近50%的地区可能无法访问Minecraft服务。

重庆重庆移动、四川成都移动、贵州贵阳移动、甘肃兰州移动、贵州贵阳联通、宁夏回族自治区银川移动、天津天津移动、浙江台州移动、西藏自治区拉萨移动、陕西西安移动、海南海口移动、上海上海移动、吉林长春移动、辽宁大连移动、安徽合肥移动、广西壮族自治区南宁移动等（复制自某视频评论区）

本人广东电信能正常访问，广东移动已被墙

**Reply by WorldHim** [reply]
浙江宁波移动试了一下，确实不行 :(
然而绍兴电信可以
估计确实是移动的问题

**Reply by zkitefly** [reply]
本人广东移动，的确被墙了

**Reply by iPodtouch6557** [reply]
补充江苏移动也寄了

**Reply by RadishTower** [reply]
银川移动联通都不行

**Comment by zkitefly** [comment]
我发现 sessionserver.mojang.com 也被墙了

`![image](https://github.com/Hex-Dragon/PCL2/assets/64117916/57f21d70-38a0-4bf1-a9d2-f1b9adec3666)`

---

Edit：

只要二级域名为 `mojang.com` 的都被墙了

`https://authserver.mojang.com`(不在启动器中使用、`https://api.mojang.com`、`https://sessionserver.mojang.com`

![https://authserver.mojang.com](https://github.com/Hex-Dragon/PCL2/assets/64117916/f7fa14e1-f7e7-4922-a360-e4ece23a318d)

![https://api.mojang.com](https://github.com/Hex-Dragon/PCL2/assets/64117916/609a13d1-cb7a-476b-83b0-6c12700b8dc5)



![https://sessionserver.mojang.com](https://github.com/Hex-Dragon/PCL2/assets/64117916/3fb50a4b-065f-481c-b417-fede417d00d6)

**Comment by SqrtTan** [comment]
有钱给lm发电但没钱买正版的离线登录路过

**Comment by kong-que-dnf123** [comment]
> 《正版登录？滚去玩网易吧》

我就知道总是会被搞出奇奇怪怪的梗图（大雾）

**Comment by kong-que-dnf123** [comment]
**截至2023/10/2 上午11:10**
被拦截的似乎只剩下移动网了
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/5e5f0899-475f-4994-87f1-bcb27d299f46)

**Comment by SALTWOOD** [comment]
目前状况：
![中国移动](https://github.com/Hex-Dragon/PCL2/assets/105980161/5071bafa-253d-491c-8381-593b1214eee3)
![中国联通](https://github.com/Hex-Dragon/PCL2/assets/105980161/164bf589-6cc5-4160-92f2-5c58ab88d236)
![中国电信](https://github.com/Hex-Dragon/PCL2/assets/105980161/db193c39-2e84-4f7b-b375-7809350818ec)
![汇总](https://github.com/Hex-Dragon/PCL2/assets/105980161/d91a39cf-0066-4ccf-bf77-ba02fbc6336b)

**Reply by iPodtouch6557** [reply]
似乎确实是的（
~连这也能环江西~

**Comment by kong-que-dnf123** [comment]
截至2023/10/3 11：47
**湖北武汉**移动与**上海**移动都已正常（状态200灰色），**新疆克拉玛依**移动也算能运行（状态为500红色）
_但是也不时会出现个别电信网因超时而失败的......_

![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/6ac8691d-94d3-4c02-b174-5037af70e550)
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/b9b608f9-c715-49a6-ad82-04dc458f156d)
